src/families/cardano/env.ts

```diff
@@ -0,0 +1,44 @@
+import { NetworkId } from "./types";
+import { types as TyphonTypes } from "@stricahq/typhonjs";
+
+import testnetCostModels from "./costmodels/testnet-protocol.json";
+
+type ENV_TYPE = {
```

Comment:
Nitpicking here, but would you agree to reword "env" with something else?
This is to avoid confusion with the global `env.ts` file and actual environment variables.

The data you're handling here closely looks like network parameters in other coins, e.g: https://github.com/LedgerHQ/ledger-live-common/blob/6aebe41709035d227c2682c8617d7f70b374ab10/src/families/bitcoin/types.ts#L76
Maybe you could set it as a `NetworkParameters` type in the types.ts file.